### PR TITLE
updated to latest HikariCP and add idea project auxiliary file to .gi…

### PR DESCRIPTION
**Pull request by xingzhefeng**
…tignore

### CODE REVIEW

**Comment by tomekw** [comment]
Thank you @xingzhefeng.

Could you, please, split `.gitignore` update and `project.clj` into two separate PRs? And also update `CHANGELOG.md` for `HikariCP` update?

File: .gitignore [hunk 1]
```diff
@@ -7,3 +7,8 @@ pom.xml.asc
 *.class
 /.lein-*
 /.nrepl-port
+/lib
+/.idea
+/.env
+*.log
+*.iml
```
Comment (tomekw) [inline .gitignore]: Misssing newline.

Comment (xingzhefeng) [inline .gitignore]: ok

**Comment by xingzhefeng** [comment]
Is it all right now ?

File: .gitignore [hunk 2]
```diff
@@ -7,3 +7,8 @@ pom.xml.asc
 *.class
 /.lein-*
 /.nrepl-port
+/lib
```
Comment (tomekw) [inline .gitignore]: Is this line needed?

File: .gitignore [hunk 3]
```diff
@@ -5,5 +5,10 @@ pom.xml
 pom.xml.asc
 *.jar
 *.class
-/.lein-*
-/.nrepl-port
+.lein-*
+.nrepl-port
+lib
```
Comment (tomekw) [inline .gitignore]: No, I meant if `lib` entry is needed here, @xingzhefeng :)

Comment (xingzhefeng) [inline .gitignore]: :) yes lib is not needed . it is my j2ee project need . You are a very strict person. verygood

**Comment by tomekw** [comment]
Now it looks great, thank you! ⭐️

**Pull request merged**